.github/workflows/release.yml

```diff
@@ -64,6 +64,17 @@ jobs:
       - name: Checkout Repo
         uses: actions/checkout@v4
 
+      - name: Determine tag from myst-to-react package.json
+        id: get_tag
+        run: |
+          VERSION=$(jq -r '.version' packages/myst-to-react/package.json)
```

Comment:
just checking my logic - this will be run *after* the changeset PR has been merged, which means that the version it finds will be the *new* version (which is what we want)